src/aosm/azext_aosm/tests/latest/integration_tests/test_nsd.py

```diff
@@ -44,7 +44,7 @@
 
 CGV_DATA = {
     "ubuntu": {
-        "nfdvName": "exampleNfdvName",
+        "nfdv": "examplenfdv",
```

Comment:
Is this correct? I thought the confusion here was that this is actually the version string (e.g. 1.0.0)

Sometimes people look at tests to work out what's happening, so we should get our tests as accurate as possible.

Same comment for lines 63, 85 and 92.

Reply:
youre right, changed